decanter_ai_sdk/client.py

```diff
@@ -520,12 +520,18 @@ def predict_ts(
         )
         return prediction
 
-    def stop_uploading(self, id):
-        logging.info(self.api.stop_uploading(id))
+    def stop_uploading(self, id: str):
```

Comment:
maybe log the id of the task that you're stopping. Do you think that'd be good?

Reply:
Yes I think that wiil be more clear!